src/mapping/config/MappingConfiguration.cpp

```diff
@@ -62,7 +62,7 @@ void addAttributes(TagStorage &storage, const std::vector<variant_t> &attributes
 // Enum required for the RBF instantiations
 enum struct RBFBackend {
   Eigen,
-  EigenGreedy,  //TODO: Das ist unschön. Aber intern die Methoder durch das Backend zu bestimmen auch!
+  EigenGreedy,  //TODO: Das ist unschön.
```

Comment:
Ideally we would have here something like `Greedy` and the exact algorithm (P or f) remains configurable.